test/backend/variables.py

```diff
@@ -44,7 +44,7 @@ def get_args_from_env():
     TEST_VERBOSE = os.getenv("TEST_VERBOSE")
     TEST_CASE_CHECK = os.getenv("TEST_CASE_CHECK")
     TEST_INVOKECONVERTER = os.getenv("TEST_INVOKECONVERTER")
-    TEST_IMPORTER_FORCE_DYNAMIC = os.getenv("TEST_IMPORTER_FORCE_DYNAMIC")
+    # TEST_IMPORTER_FORCE_DYNAMIC = os.getenv("TEST_IMPORTER_FORCE_DYNAMIC")
```

Comment:
Looks like `TEST_IMPORTER_FORCE_DYNAMIC` is no longer used. If you don't mind removing it completely.

Reply:
Deleted the line. Thanks!

Reply:
Thanks!